galeracluster/source/documentation/sst-physical.rst

```diff
@@ -125,24 +125,33 @@ For more information on ``xtrabackup``, see the `Percona XtraBackup User Manual
 .. rst-class:: section-heading
 .. rubric:: ``clone``
 
-Starting with version 8.0.22 ``clone`` SST method is available for Galera
-CLuster for MySQL. It is based on the native MySQL clone plugin. It
+Starting with version 8.0.22, the ``clone`` SST method is available for Galera
+Cluster for MySQL. It is based on the native MySQL clone plugin. It
 proved to be much faster than ``xtrabackup``, however it will block Donor
-node on DDL execution if that happens during the transfer.
+node on DDL execution, if that happens during the transfer.
 
-Basic configuraition for ``clone`` SST on Joiner:
+Basic configuration for ``clone`` SST on Joiner:
 
 .. code-block:: ini
 
     [mysqld] 
     wsrep_sst_method=clone
 
-Basic configuraition for ``clone`` SST on Donor:
+Grant privileges on Donor:
+
+.. code-block:: ini
+
+    CREATE USER 'sst'@'%' IDENTIFIED BY 'secret';
+    GRANT BACKUP_ADMIN, CLONE_ADMIN ON . TO 'sst'@'localhost';
+    GRANT SYSTEM_VARIABLES_ADMIN ON . TO 'sst'@'localhost';
+
+
+Basic configuration for ``clone`` SST on Donor:
 
 .. code-block:: ini
 
     [mysqld]
-    wsrep_sst_auth=<admin user>:<admin password>
+    wsrep_sst_auth=some_user:some_password
```

Comment:
I'm afraid I was understood too literally.
```
wsrep_sst_auth=<admin user>:<admin password>
```
was supposed to mean that a `<user>:<password>` pair should be given, with the user having sufficient administrative privileges like:
1. the user should be able to connect via localhost
2. the user should have BACKUP_ADMIN, CLONE_ADMIN, SYSTEM_VARIABLES_ADMIN on all of the database

Above is an example how to create such user (although we could probably do with a single GRANT statement)

Just to be make sure it is understood:
Above is an example how to create MySQL user for CLONE SST: in the example the user is called `sst` and is given the password `secret`. And granted required privileges. Then this user and password should be put in `wsrep_sst_auth`:
```
wsrep_sst_auth=sst:secret
```
Just the same you could use a root user, which already exists:
```
wsrep_sst_auth=root:<root password>
```
Using in the user creation example `sst` and in the configuration example `some_user` does not really connect the dots